requirements.txt

```diff
@@ -1,5 +1,5 @@
 django>=1.8,<1.9
-djangorestframework>=3.1,<3.2
+git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
```

Comment:
The reason why is documented here:
https://openedx.atlassian.net/browse/PLAT-1581?focusedCommentId=270321&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-270321
It'd be *great* to use the stable release again - perhaps we can get to that point soon.